web/index.ts

```diff
@@ -30,10 +30,34 @@ import "reflect-metadata"; //leave it here...
 import { apolloServer } from "./../graphql";
 import rateLimit from "express-rate-limit";
 import * as notificationController from "./controllers/notificationController";
+import { isDev } from "../common/util/environment";
+
 
 // Logger setup
 try {
-    configure("web/logconfig.json");
```

Comment:
Is there a particular reason for this not being in a separate file anymore @Jonasdoubleyou?

Reply:
Yeah, wanted to change the config dependent on `process.env.ENV`, so that it's enabled in our dev landscape but not on PROD. That's only possible with config "as code".

Reply:
I actually tried to move this to /common/ but then noticed that we have a different logger config for the jobs, so this would require quite some refactoring to align ...